quix-frontend/client/src/lib/sql-autocomplete/adapter/trinoToJs.ts

```diff
@@ -0,0 +1,146 @@
+const enum StartingOptions {
+  Row = 'row(',
+  Map = 'map(',
+  Array = 'array(',
+  TimeStamp = 'timestamp(',
+}
+const enum SpecialCharacters {
+  OpenParenthesis = '(',
+  ClosedParenthesis = ')',
+  Comma = ',',
+  Space = ' ',
+}
+
+export function trinoToJs(trinoObjectAsString: string, index: number):  object|string {
+  if (trinoObjectAsString.startsWith(StartingOptions.Row , index)) {
+    return processRow(trinoObjectAsString, index + StartingOptions.Row.length);
+  }
+  if (trinoObjectAsString.startsWith(StartingOptions.Map, index )) {
+    return 'map';
+  }
+
+  if ( trinoObjectAsString.startsWith(StartingOptions.Array, index ) ) {
+    return 'array';
+  }
+    return trinoObjectAsString;
+}
+
+export function processRow(trinoObjectAsString: string, index: number): object {
+  const finalObject: object = {};
+  let key = "";
+  let value = "";
+  let objectToInsert;
+  let isFirstWord = true;
+  const lengthOfSubstring = 5;
```

Comment:
magic number?

Reply:
would you prefer i use it as an enum?
i wrote it at the  beging of the code like this because i though it would look better then floating 5 around the code

Reply:
it should be a constant, with a name like THIS_NAME as we are writing constants in all languages

Reply:
fixed:

# chen ulmar